Content.Server/Communications/CommunicationsConsoleSystem.cs

```diff
@@ -328,6 +344,36 @@ private void OnRecallShuttleMessage(EntityUid uid, CommunicationsConsoleComponen
             _roundEndSystem.CancelRoundEndCountdown(uid);
             _adminLogger.Add(LogType.Action, LogImpact.High, $"{ToPrettyString(message.Actor):player} has recalled the shuttle.");
         }
+
+        // Begin DeltaV - Exfiltration Shuttle
+        private void OnCallExfiltrationMessage(Entity<CommunicationsConsoleComponent> ent, ref CommunicationsConsoleCallExfiltrationShuttleMessage args)
+        {
+            if (_stationSystem.GetOwningStation(ent) is not { } station)
+                return;
+
+            if (!CanUse(args.Actor, ent))
+            {
+                _popupSystem.PopupEntity(Loc.GetString("comms-console-permission-denied"), ent, args.Actor);
+                return;
+            }
+
+            _stationExfiltration.Call(station);
+        }
+
+        private void OnRecallExfiltrationMessage(Entity<CommunicationsConsoleComponent> ent, ref CommunicationsConsoleRecallExfiltrationShuttleMessage args)
+        {
+            if (_stationSystem.GetOwningStation(ent) is not { } station)
+                return;
+
+            if (!CanUse(args.Actor, ent))
+            {
+                _popupSystem.PopupEntity(Loc.GetString("comms-console-permission-denied"), ent, args.Actor);
+                return;
+            }
+
+            _stationExfiltration.Recall(station);
+        }
+        // End DeltaV - Exfiltration Shuttle
     }
```

Comment:
1. make CanUse public
2. move this to a partial system
3. combine the messages and have a bool for recalling
these are identical except for the function called at the end